app/routes/circulars/index.tsx

```diff
@@ -35,6 +36,9 @@ const limit = 100
 export async function loader({ request: { url } }: DataFunctionArgs) {
   const { searchParams } = new URL(url)
   const query = searchParams.get('query') || undefined
+  if (query) {
+    await circularRedirect(query.toString())
```

Comment:
It's already guaranteed to be a string by this point.

```suggestion
    await circularRedirect(query)
```

Reply:
oh yeah that was a holdover from when I was trying to figure out the issue with parseFloat